ytdownloader/converter.py

```
@@ -0,0 +1,159 @@
+
+            ydl_opts = {
+                'format': 'bestaudio/best' if to_mp3 else 'bestvideo[ext=mp4]+bestaudio[ext=m4a]/best[ext=mp4]/best',
+                'outtmpl': str(output_folder / f"%(id)s.{'mp3' if to_mp3 else 'mp4'}"),
+                'default_search': 'auto',  # Set default search
+                'progress_hooks': [self.my_hook],  # Add progress hook
+            }
+
+            # Always add cookies option
+            ydl_opts['cookies'] = str(self.data_folder / "cookies.txt")  # Specify the path to your cookies file
+
+            conversion_message = await ctx.send(f"`Converting video...`")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)